CHANGELOG.md

```diff
@@ -1,72 +1,78 @@
-v1.3.0
-
-15-11-2017
-
-* Pass options from index.js to the spellChecker
-* Fix Jekyll front matter being spellchecked if it contained a regular expression (#107)
-* Fix content between Jekyll front matter and in-content headings not being spellchecked
-
-v1.2.0
-
-12-11-2017
-
-* Only ignore test between two horizontal rules if it looks like Jekyll front matter
-
-v1.1.0
-
-12-11-2017
-
- * Allow markdown relative spelling files
-
-v1.0.0
-
-05-08-2017
-
- * Added support for Australian English
- * Upgraded packages
- * Dropped support for node < 4
-
-v0.11.0
-
-03-06-2016
-
- * Support for cryllic characters
-
-v0.10.0
-
-17-03-2016
-
- * Upgrade English GB dictionary
- * Support for Kramdown style code blocks
-
-v0.9.1
-
-12-03-2016
-
- * Support for additional UTF characters
-
-v0.9.0
-
-09-01-2016
-
- * Fix issue correcting words when filtering
- * Add support for negated patterns
-
-v0.8.0
-
-04-01-2016
-
- * Upgrade Babel and fix bad ES6 import
- * Fix showing bad spelling report at end of processing
- * Fix allowing double backtick in markdown
- * Fix saving words with `\u2019` apostrophe
- * Upgrade English GB dictionary
-
-v0.7.0
-
-27-12-2015
-
- * Upgrade dependencies
- * Upgrade English GB dictionary
- * Add Spanish dictionary
-
+v1.3.1
```

Comment:
I don't think we need to change all the line endings in this changelog? Just the bin needs to be published with LFs (which is what it has in the repo I expect)